verifact/forum/models.py

```diff
@@ -10,3 +11,23 @@ class Question(models.Model):
 
     def __str__(self):
         return self.text
+
+
+class Answer(models.Model):
+    class AnswerTypes(models.IntegerChoices):
+        TRUE = 1, _("True")
+        FALSE = 2, _("False")
+        NEITHER = 3, _("Neither")
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    answer = models.IntegerField(choices=AnswerTypes.choices)
+    text = models.CharField(max_length=2048)
+    citation_url = models.CharField(max_length=2048)
+    citation_title = models.CharField(max_length=2048)
+    credible_count = models.IntegerField()
+    not_credible_count = models.IntegerField()
+    question = models.ForeignKey(Question, on_delete=models.CASCADE)
```

Comment:
Ah sorry, `related_name` is actually an argument to `ForeignKey`. See the docs here: https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ForeignKey.related_name